person.rb

```diff
@@ -10,6 +11,7 @@ def initialize(age, parent_permission: true, name: 'Unknown')
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
```

Comment:
- [ ] The `Book` and `Person` classes have a **_many-to-many_** relationship through the `Rental` class. This can be successfully implemented by using the `@rentals` array and the `add_rental` methods in both the `Book` and `Person` classes. This would enable their instances to keep records of all the rentals that belong to that instance. 👍 😎 

I would advise you to create the `add_rental` method in the `Person` class to successfully implement the relationship between both classes. The `add_rental` method should create a new instance of the `Rental` class. 💻 🖱️ 

#### Hint:

```ruby
def add_rental(date, book)
  Rental.new(date, book, self)
end
```